Extension/src/LanguageServer/client.ts

```diff
@@ -1414,6 +1414,9 @@ export class DefaultClient implements Client {
                 references: {
                     ...vscode.workspace.getConfiguration("C_Cpp.references", this.RootUri)
                 },
+                tagParseAsC: {
```

Comment:
This is not required. This would only be used if "tagParseAsC" had additional settings on the right of a ".". Also, the setting isn't being passed to the language server during initialization.